#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_no-depmod.dpatch by Ian Campbell <ijc@hellion.org.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Do not run depmod on install

@DPATCH@

--- ivtv-0.6.2.orig/driver/Makefile
+++ ivtv-0.6.2/driver/Makefile
@@ -35,7 +37,6 @@
 endif
 
 install::
-	/sbin/depmod -a
 	@echo
 	@echo "v4l-cx2341x-init.mpg needs copying to the hotplug firmware"
 	@echo "directory if needed for PVR350 mpeg initialization"
